.nvmrc

```diff
@@ -0,0 +1,2 @@
+stable
```

Comment:
Should peg to a known working version so this doesn't unexpectedly bump major versions and break stuff.